### PR TITLE
Dockerfile - Upgrade to rocm5.7 dockerfile

### PR DESCRIPTION
**Pull request by yukirora**
**Description**
upgrade to rocm5.7 dockerfile.

### CODE REVIEW

File: .github/workflows/build-image.yml [hunk 1]
```diff
@@ -30,26 +30,9 @@ jobs:
         - name: cuda11.1.1
```
Comment (cp5555) [inline .github/workflows/build-image.yml:30]: Please revise document.

**Review by cp5555 — APPROVED** [review]

File: .github/workflows/build-image-rocm.yml [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+    runs-on: [self-hosted, build]
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        include:
+        - name: rocm5.7
+          dockerfile: rocm5.7.x
+          tags: superbench/main:rocm5.7
```
Comment (abuccts) [inline .github/workflows/build-image-rocm.yml:29]: can you merge with previous yml?

File: superbench/benchmarks/micro_benchmarks/gpu_copy_performance/CMakeLists.txt [hunk 3]
```diff
@@ -1,38 +1,38 @@
...
+ 
 cmake_minimum_required(VERSION 3.18)
-
+ 
 project(gpu_copy LANGUAGES CXX)
-
+ 
 find_package(CUDAToolkit QUIET)
-
+
```
Comment (abuccts) [inline superbench/benchmarks/micro_benchmarks/gpu_copy_performance/CMakeLists.txt:9]: same

File: .github/workflows/build-image-rocm.yml [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+    - release/*
+  release:
+    types:
+    - published
+  workflow_dispatch:
+
+jobs:
+  docker:
+    name: Docker build ${{ matrix.name }}
+    runs-on: [self-hosted, build]
```
[the remark this answers is not in the document]
Comment (yukirora) [reply .github/workflows/build-image-rocm.yml]: ok, updated

File: .github/workflows/build-image-rocm.yml (hunk 2 above)
Comment (yukirora) [reply .github/workflows/build-image-rocm.yml]: sure

**Pull request merged**